reinforcement_learning/rl_on_gcp_demo/trainer/td3_agent.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Implementation of a TD3 agent.
+
+Implementation of TD3 - Twin Delayed Deep Deterministic Policy Gradient
+Algorithm and hyperparameter details can be found here:
+https://arxiv.org/pdf/1802.09477.pdf
+
+"""
+import agent
+from common import replay_buffer
+from common.actor_critic import ActorNetwork, CriticNetwork
+import numpy as np
+
+
+class TD3(agent.Agent):
+    """TD3 agent."""
+
+    def __init__(self, env, sess, config):
+        """Initialize members."""
+        self.env = env
```

Comment:
nit: Move self.env one line below to separate: Class variables and method variables.

Reply:
fixed in ac35cd106aa30d9f4e0cc69898e7da768c8518a7